### ISSUE TITLE
Can't install integration with HA 2023.12.1

### ISSUE THREAD

**Issue by reedstrm** [music-assistant/hass-music-assistant#1698]
### What version of Music Assistant has the issue?

2023.6.0b6

### The problem

I get an popup with error `Config flow could not be loaded: {"message":"Invalid handler specified"}` when trying to add the integration.

Logs contain the error "no such module music_assistant". Perhaps related to the dependabot PR #1688 for upgrading to 2023.12.1 which is a failure to import async_timeout

### How to reproduce

Attempt to install on 2023.12.1

### Relevant log output

2023-12-14 01:05:41.399 ERROR (MainThread) [homeassistant.config_entries] Error occurred loading flow for integration mass: No module named 'music_assistant'

### What version of Home Assistant Core are your running

2023.12.1

### What type of installation are you running?

Docker Container

### On what type of hardware are you running?

Generic x86-64 (e.g. Intel NUC)

**Comment by OzGav** [comment]
Do you have the addon installed?

**Comment by reedstrm** [comment]
I know I selected docker container, somehow it got reverted to HA OS. I have the server installed on a different PC, independently. This may be a HACS/docker issue. I just opened a bash shell in the correct container and used pip to hand install the required version:  music-assistant==2.0.0b59 and the integration came up. I'll close this and go hunt up the HACS repo to see if there are any similar issues.
